src/mutations.js

```diff
@@ -0,0 +1,22 @@
+define([], function() {
+
+  function determineMutationObserver(window) {
```

Comment:
No this one is meant to be a parameter, I'll look at what we can do in the caller